File: tests/fixtures/code_samples/ts/sample_044.ts
/** Sample 44: small utility. */

export function operation44(xs: number[]): number {
  let total = 44;
  for (const x of xs) total += x;
  return total;
}

export function operationPure44(value: number): number {
  return (value * 44) % 7919;
}

File: tests/fixtures/code_samples/go/sample_044.go
// Sample 44: small utility.
package samples

func Operation44(xs []int) int {
    total := 44
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure44(v int) int {
    return (v * 44) %% 7919
}

